source/deploy/postgres-migration.rst

```diff
@@ -193,6 +193,33 @@ Ensure that the ``mmuser`` user in PostgreSQL is the owner of the schema.
 
 Then, re-run the command from step 2.
 
+Incoming webhook channel names become case-sensitive after migration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After migrating from MySQL to PostgreSQL, incoming webhooks that previously worked with mixed-case channel names (e.g., ``"Tech-Talk"``) will fail with the following error:
+
+.. code-block:: text
+
+  GetChannelByName: Channel does not exist.
+
+This occurs because Mattermost does not allow creating channels with uppercase names - all new channels must have lowercase names.
```

Comment:
```suggestion
This occurs because Mattermost doesn't allow the creation of channels with uppercase characters. All new channels must only contain lowercase characters.
```